### DISCUSSION TITLE
DDC not working on MacBook Pro M1 even directly connected via HDMI

### DISCUSSION THREAD

**Discussion by bverkron** [MonitorControl/MonitorControl#1460] [Q&A]
Was working perfectly fine on my MacBook Pro 2014 via the full sized HDMI port but now that I've upgraded to a MacBook Pro M1 it only seems to control it via software even when plugged into the full HDMI port and despite it saying DDC is enabled.

![image](https://github.com/MonitorControl/MonitorControl/assets/19174503/a6f42b19-90ef-4010-a502-5f4964ea65cf)

I see https://github.com/MonitorControl/MonitorControl/discussions/1456 but it seems to only apply to USB-C docks / hubs.

Suggestions on what to try?

**Comment by waydabber** [comment]
MonitorControl does not support the HDMI port of any M1 Macs. Try [BetterDisplay](https://betterdisplay.pro) instead as it has hardware DDC support for all connection types.

**Comment by bverkron** [comment]
Ah, I did not see that in the read me exceptions. Thought I searched there. Thanks.
